common/djangoapps/util/password_policy_validators.py

```diff
@@ -1,322 +1,471 @@
 """
-This file exposes a number of password complexity validators which can be optionally added to
+This file exposes a number of password validators which can be optionally added to
 account creation
-
-This file was inspired by the django-passwords project at https://github.com/dstufft/django-passwords
-authored by dstufft (https://github.com/dstufft)
 """
-from __future__ import division
+from __future__ import unicode_literals
 
 import logging
-import string
 import unicodedata
 
 from django.conf import settings
+from django.contrib.auth.password_validation import (
+    get_default_password_validators,
+    CommonPasswordValidator,
+    MinimumLengthValidator,
+    UserAttributeSimilarityValidator,
+)
 from django.core.exceptions import ValidationError
-from django.utils.translation import ugettext_lazy as _
-from django.utils.translation import ungettext_lazy as ungettext
-from Levenshtein import distance
+from django.utils.translation import ugettext as _, ungettext
 from six import text_type
 
 from student.models import PasswordHistory
 
-
 log = logging.getLogger(__name__)
 
-# In description order
-_allowed_password_complexity = [
-    'ALPHABETIC',
-    'UPPER',
-    'LOWER',
-    'NUMERIC',
-    'DIGITS',
-    'PUNCTUATION',
-    'NON ASCII',
-    'WORDS',
-]
 
-
-class SecurityPolicyError(ValidationError):
-    pass
+def password_validators_instruction_texts(password_validators=None):
+    """
+    Return a string of instruction texts of all configured validators.
+    Expects at least the MinimumLengthValidator to be defined.
+    """
+    complexity_instructions = []
+    # For clarity in the printed instructions, the minimum length instruction
+    # is separated from the complexity instructions. The substring is used as
+    # an indicator to find the proper instruction.
+    length_instruction = ''
+    length_instruction_substring = 'at least'
```

Comment:
This is a brittle way to check for min length. In particular, it will fail on any non-English language.

Can you just check "isinstance(validator, MinimumLengthValidator)"?